pkg/kv/dist_sender.go

```diff
@@ -59,6 +60,39 @@ const (
 	defaultSenderConcurrency = 500
 )
 
+var (
+	// metaDistSenderTotals identify the total rpc sends of DistSender.
+	metaDistSenderTotals = metric.Metadata{Name: "distsender.totals"}
```

Comment:
thanks for the additional comments. We still need a `help` field in the metadata though, that's what is shown in the `/_status/vars` page and avoids having to dig through the code.

I could like to suggest the following names instead:
* distsender.rpc.sent
* distsender.rpc.sent-local (the naming makes it clear this is a subset of `sent`)
* distsender.rpc.retries (I'm assuming this is a subset of `sent`?)
* distsender.rpc.errors.notleaseholder (is this a subset of `sent`?)